[32mINFO    [0m integration.ha.test_backups:test_backups.py:490 Remove s3-credentials relation
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:502 Re-add s3-credentials relation
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [executing] maintenance: Configuring backup service...
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:511 Syncing credentials for aws
[32mINFO    [0m juju.model:model.py:2958 Waiting for model:
  main/0 [idle] active: 
  s3-integrator/0 [idle] active: 
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.151.38.162:9200/_recovery?human
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=368, count=369, last_expected_id=368)
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.151.38.162:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.151.38.162:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.151.38.162:9200/series_index/_count
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.151.38.162:9200/_recovery?human
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.151.38.162:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.151.38.162:9200/series_index/_count
[32mINFO    [0m integration.ha.helpers:helpers.py:591 Testing restore for series_index - original count pre-restore: 369, and now, new count: 331
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.151.38.162:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.151.38.162:9200/series_index/_count
[32mINFO    [0m integration.ha.conftest:conftest.py:39 



The writes have been cleared.




[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.4.4    unsupported  02:30:56Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active      2  opensearch                                 0  no       
main                               active      1  opensearch                                 2  no       
opensearch                         active      1  opensearch                                 1  no       
s3-integrator                      active      1  s3-integrator             latest/edge     31  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
failover/0                   active    idle   2        10.151.38.23    9200/tcp  
failover/1*                  active    idle   3        10.151.38.38    9200/tcp  
main/0*                      active    idle   4        10.151.38.162   9200/tcp  
opensearch/0*                active    idle   5        10.151.38.96    9200/tcp  
s3-integrator/0*             active    idle   0        10.151.38.90              
self-signed-certificates/0*  active    idle   1        10.151.38.15              

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.151.38.90   juju-d4e164-0  ubuntu@22.04      Running
1        started  10.151.38.15   juju-d4e164-1  ubuntu@22.04      Running
2        started  10.151.38.23   juju-d4e164-2  ubuntu@22.04      Running
3        started  10.151.38.38   juju-d4e164-3  ubuntu@22.04      Running
4        started  10.151.38.162  juju-d4e164-4  ubuntu@22.04      Running
5        started  10.151.38.96   juju-d4e164-5  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-1: 02:09:25 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 02:09:25 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 02:09:25 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-0: 02:09:27 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 02:09:27 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-s3-integrator-0: 02:09:27 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-4: 02:10:05 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 02:10:05 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-5: 02:10:05 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-5: 02:10:05 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-main-0: 02:10:05 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-opensearch-0: 02:10:05 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-3: 02:10:08 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 02:10:08 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-1: 02:10:08 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 02:10:18 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 02:10:18 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-0: 02:10:19 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-opensearch-0: 02:14:58 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:14:59 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:14:59 ERROR unit.opensearch/0.juju-log upgrade-version-a:9: [Errno 111] Connection refused
unit-failover-1: 02:15:00 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:00 ERROR unit.main/0.juju-log peer-cluster-orchestrator:10: [Errno 111] Connection refused
unit-opensearch-0: 02:15:00 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:15:01 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:01 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:01 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:01 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:15:02 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 02:15:02 ERROR unit.main/0.juju-log peer-cluster-orchestrator:10: [Errno 111] Connection refused
unit-opensearch-0: 02:15:02 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:02 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:02 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 02:15:02 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:15:03 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:03 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-main-0: 02:15:03 ERROR unit.main/0.juju-log peer-cluster-orchestrator:10: [Errno 111] Connection refused
unit-failover-1: 02:15:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:15:04 ERROR unit.failover/0.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-main-0: 02:15:05 ERROR unit.main/0.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-1: 02:15:05 ERROR unit.failover/1.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-failover-0: 02:15:05 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-main-0: 02:15:07 ERROR unit.main/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-main-0: 02:15:07 ERROR unit.main/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-main-0: 02:15:07 ERROR unit.main/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-main-0: 02:15:07 ERROR unit.main/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 02:15:07 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:15:07 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-main-0: 02:15:08 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-opensearch-0: 02:15:08 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:15:08 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-1: 02:15:08 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:15:09 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:09 ERROR unit.main/0.juju-log peer-cluster-orchestrator:10: [Errno 111] Connection refused
unit-failover-0: 02:15:09 ERROR unit.failover/0.juju-log node-lock-fallback:2: [Errno 111] Connection refused
unit-failover-1: 02:15:09 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:09 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:09 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:10 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:10 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:15:10 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:15:10 ERROR unit.failover/0.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-failover-1: 02:15:11 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-main-0: 02:15:11 ERROR unit.main/0.juju-log peer-cluster-orchestrator:10: [Errno 111] Connection refused
unit-opensearch-0: 02:15:11 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:15:12 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-main-0: 02:15:13 ERROR unit.main/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-opensearch-0: 02:15:14 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-main-0: 02:15:15 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-opensearch-0: 02:15:16 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-1: 02:15:17 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:17 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-0: 02:15:19 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:15:19 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:19 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:15:20 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:15:20 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:20 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:20 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:15:21 ERROR unit.failover/1.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-failover-0: 02:15:21 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-opensearch-0: 02:15:22 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:22 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:22 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:22 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 02:15:22 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:15:22 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:15:22 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:23 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:15:23 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-main-0: 02:15:23 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:15:23 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:24 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:15:24 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 02:15:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:15:24 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:15:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:25 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:25 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:15:25 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:25 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-main-0: 02:15:25 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:25 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 02:15:25 ERROR unit.failover/0.juju-log node-lock-fallback:2: [Errno 111] Connection refused
unit-main-0: 02:15:25 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:26 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:15:26 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-main-0: 02:15:26 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:26 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:15:26 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:26 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:26 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:26 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 02:15:26 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:15:26 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:27 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-main-0: 02:15:29 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:30 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-main-0: 02:15:33 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:34 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:15:34 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-main-0: 02:15:37 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:38 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 02:15:40 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-main-0: 02:15:41 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:41 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-main-0: 02:15:44 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:45 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:15:48 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:48 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:48 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:15:48 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 02:15:49 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:15:49 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:50 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:50 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:50 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:15:50 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 02:15:50 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-main-0: 02:15:51 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:51 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:15:51 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:52 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 02:15:52 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:15:53 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:15:53 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 02:15:53 ERROR unit.failover/1.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-main-0: 02:15:54 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:54 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 02:15:54 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:15:55 ERROR unit.failover/1.juju-log node-lock-fallback:2: [Errno 111] Connection refused
unit-failover-0: 02:15:56 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:15:56 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:15:57 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:57 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:15:57 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:15:58 ERROR unit.failover/1.juju-log node-lock-fallback:2: [Errno 111] Connection refused
unit-failover-0: 02:15:58 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:15:58 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:15:58 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:15:58 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:15:58 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:15:58 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:15:58 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:00 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-main-0: 02:16:00 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:16:00 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-0: 02:16:00 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:16:01 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-0: 02:16:02 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-main-0: 02:16:03 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:16:03 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:16:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:16:04 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 02:16:04 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:04 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:04 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:04 ERROR unit.failover/0.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-failover-1: 02:16:05 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-opensearch-0: 02:16:05 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:05 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:05 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:05 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 02:16:06 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:06 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:06 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:16:06 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:16:06 ERROR unit.main/0.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:16:17 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:17 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:23 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 02:16:23 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:16:23 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:16:24 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:24 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:24 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 02:16:24 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:16:24 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 02:16:24 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:24 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:16:25 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:16:26 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:26 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:26 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:26 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 02:16:28 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:16:28 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:16:28 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:16:29 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:16:29 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:16:29 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:16:29 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:29 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:29 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 02:16:29 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:16:29 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 02:16:30 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:16:30 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:16:31 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:31 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:31 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:31 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:32 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:33 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:33 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:33 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:33 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:33 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 02:16:33 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:33 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:33 ERROR unit.failover/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-0: 02:16:33 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:33 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:33 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:34 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:34 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:34 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:34 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:34 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:34 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:34 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:34 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:34 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:34 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:34 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:34 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:34 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:34 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:34 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:35 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:35 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:35 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:35 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:35 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:35 ERROR unit.failover/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-0: 02:16:35 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:36 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:36 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:16:36 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:36 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 02:16:36 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:37 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:37 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:39 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:42 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:45 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:16:46 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:47 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:47 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:16:47 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-0: 02:16:49 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:53 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:56 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:16:59 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:59 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:16:59 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:16:59 ERROR unit.failover/1.juju-log peer-cluster:10: Cannot connect to the OpenSearch server...
unit-failover-1: 02:16:59 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:16:59 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:00 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:00 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:01 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:01 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:17:01 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:17:01 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:17:01 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:17:01 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 02:17:02 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:02 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:17:02 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:02 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:03 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-opensearch-0: 02:17:03 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:17:04 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:04 ERROR unit.failover/1.juju-log peer-cluster:10: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:04 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:04 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:04 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:04 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:05 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:05 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:17:05 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:05 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:05 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:06 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:06 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:17:08 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:11 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:14 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:17 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:21 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:23 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:23 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:23 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:23 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:24 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:24 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:24 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:24 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:25 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:25 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:25 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:25 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:26 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:26 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:17:27 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:28 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:28 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:28 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:28 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:17:28 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:17:28 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:17:28 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:17:28 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-failover-1: 02:17:29 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 02:17:29 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:35 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:38 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:45 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:48 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:51 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:54 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:17:57 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:18:00 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:18:03 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:18:06 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:18:09 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:18:12 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 02:18:22 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:18:23 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:18:24 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:18:24 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 02:18:26 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:18:27 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:28 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:28 ERROR unit.opensearch/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:18:28 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:29 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:29 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:29 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:30 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:31 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:31 ERROR unit.opensearch/0.juju-log opensearch-peers:7: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:18:31 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:35 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:38 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:44 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:47 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:51 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:54 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:18:57 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:19:00 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:19:03 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:19:06 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:19:10 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:19:13 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-opensearch-0: 02:19:16 ERROR unit.opensearch/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 02:21:08 ERROR unit.main/0.juju-log s3-credentials:16: Plugin repository-s3 missing: ['access-key', 'secret-key']
unit-main-0: 02:21:11 ERROR unit.main/0.juju-log s3-credentials:16: Plugin repository-s3 missing: ['access-key', 'secret-key']
unit-main-0: 02:21:14 ERROR unit.main/0.juju-log s3-credentials:16: Plugin repository-s3 missing: ['access-key', 'secret-key']
unit-main-0: 02:21:26 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:28 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:29 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:30 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:32 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:33 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-main-0: 02:21:49 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:51 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:52 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:54 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:55 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:21:56 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-main-0: 02:22:15 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:22:17 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:22:19 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:22:21 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:22:23 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-failover-1: 02:22:37 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:22:37 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 02:22:39 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:22:39 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 02:22:41 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:22:42 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 02:22:43 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:22:43 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 02:22:45 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:22:45 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-main-0: 02:22:47 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-failover-1: 02:22:47 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 02:22:48 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-main-0: 02:22:49 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:22:51 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:22:54 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:22:56 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:26:54 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:26:56 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:26:57 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:26:59 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:27:01 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:27:22 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:27:23 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:27:25 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:27:27 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository
unit-main-0: 02:27:29 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.151.38.162:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'https://s3.amazonaws.com', 'protocol': 'https', 'bucket': 'data-charms-testing', 'base_path': 'opensearch/f8331219-7241-4ed5-a4cb-aaeb971eb247', 'region': 'us-east-1'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.151.38.162:9200/_snapshot/s3-repository

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...
[32mINFO    [0m integration.ha.test_backups:test_backups.py:173 Cleaning backups from cloud buckets